master/internal/command/manager.go

```diff
@@ -38,10 +38,10 @@ func respondBadRequest(ctx *actor.Context, err error) {
 
 // parseCommandRequest parses an API request from the following components:
 //
-// - config: the command configuration
-// - template: the configuration template name
-// - user_files: the files to run with the command
-// - data: additional data for a command
+// - config: The command configuration.
+// - template: The configuration template name.
+// - user_files: The files to run with the command.
+// - data: Additional data for a command.
```

Comment:
thought: I think this linter is over the top here -- we shouldn't need to make these changes. I would recommend we disable `godot` since this formatting is a bit too restrictive on us here

Reply:
I did feel a _bit_ silly changing these, but I think making sure the 99% case (i.e., regular sentences) follows the clear convention is worth it. (Also, to be clear, the check itself is only that there's one final `.` at the end; everything else is me doing stuff because I felt like it.)

Reply:
I agree with @brain-good that this is over the top as well. I would prefer we focus our efforts on linting rules that have a more consequential impact.